tests/lax_numpy_test.py

```diff
@@ -1252,6 +1252,40 @@ def testCumSumProd(self, axis, shape, dtype, out_dtype, onp_op, jnp_op, rng_fact
                             tol=tol)
     self._CompileAndCheck(jnp_fun, args_maker, check_dtypes=True)
 
+
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_yshape={}_xshape={}_dx={}_axis={}".format(
+        jtu.format_shape_dtype_string(yshape, dtype),
+        jtu.format_shape_dtype_string(xshape, dtype) if xshape is not None else None,
+        dx,
+        axis),
+        "yshape": yshape,
+        "xshape": xshape,
+        "dtype": dtype,
+        "dx": dx,
+        "axis": axis,
+        "rng_factory": jtu.rand_default}
```

Comment:
There's no particular reason to put `rng_factory` as a test case argument any more unless you want to test more than one distribution.

In the past, we had this convention because `rng` was a stateful object and we wanted to initialize it once and use it in all the test cases. However, we moved away from that convention so tests are deterministic and order-independent.

Reply:
Done.